lib/robots/roborock/RoborockValetudoRobot.js

```diff
@@ -100,20 +100,6 @@ class RoborockValetudoRobot extends MiioValetudoRobot {
                     }
                 });
                 return true;
-            case "_sync.gen_tmp_presigned_url":
```

Comment:
We can and must keep roborock as-is, since they don't use the common format for the `_sync.gen_presigned_url` response

Reply:
I see. In that case, I can just override it and call the parent method on no match. Or just don't call it.

Reply:
I reverted the changes for the roborock robot.